GameServer/packets/Server/PacketLib1126.cs

```diff
@@ -92,9 +102,10 @@ public override void SendCharacterOverview(eRealm realm)
 
 			int firstSlot = (byte)realm * 100;
 
```

Comment:
Cast to `uint` immediately with `u`suffix after the number.

Reply:
I understand the decision here, since the value can at no point overflow. That being said however, this is unrelated to this PR.

Reply:
Ok wrong line sorry. The comment was meant to apply to line 105 below with the realm bit.

Reply:
It is not enough to warrant a new commit, but it is indeed nicer.